### PR TITLE
Add CHASM data to config

### PR DESCRIPTION
**Pull request by evetion**

### CODE REVIEW

File: dgds_backend/config_data/datasets.json [hunk 1]
```diff
@@ -16,8 +16,207 @@
     }
   ],
   "datasets": [
+    {
+      "bbox": [[3.0, 50.0], [7.0, 57.0]],
+      "scope": "regional",
```
Comment (ckrogers) [inline dgds_backend/config_data/datasets.json]: Does the scope belong in each dataset? or each theme?

File: dgds_backend/config_data/datasets_access.json [hunk 2]
```diff
@@ -1,4 +1,106 @@
 {
+  "chasm_wvh": {
+    "dataService":  {
+      "url": "http://pl-tc012.xtr.deltares.nl:8080/FewsWebServices/rest/digitaledelta/2.0",
+      "name": "",
```
Comment (ckrogers) [inline dgds_backend/config_data/datasets_access.json]: Need the data id, or do we not have these available yet?

File: dgds_backend/config_data/datasets.json (hunk 1 above)
Comment (evetion) [reply dgds_backend/config_data/datasets.json]: What a theme is isn't defined yet (theme, region, dataset). So this allows us to implement something that works, even if there are overlapping themes. A new theme has been set, which could at a later point be "scoped" as well. For now, this scope allows zooming functionality.

File: dgds_backend/config_data/datasets_access.json (hunk 2 above)
Comment (evetion) [reply dgds_backend/config_data/datasets_access.json]: Good one, will ask and update.

**Pull request merged**